app/scripts/lib/constants.js

```diff
@@ -52,7 +52,10 @@ define([], function () {
     ACCOUNT_UPDATES_WEBCHANNEL_ID: 'account_updates',
 
     VERIFICATION_REDIRECT_ALWAYS: 'always',
-    VERIFICATION_REDIRECT_NO: 'no'
+    VERIFICATION_REDIRECT_NO: 'no',
+
+    MARKETING_EMAIL_OAUTH_SCOPE: 'basket:write profile:email',
```

Comment:
This constant is only being used by one module– perhaps we can make it a local constant?